src/components/Footer/Footer.js

```diff
@@ -1,26 +1,53 @@
-import './Footer.css'
+import styled from 'styled-components'
 
 function Footer () {
     return (
         <footer>
-            <nav className="footer__nav">
-                <button className="footer__button">
+            <FooterContainer>
+                <FooterButton>
                 Home
         
-                </button>
+                </FooterButton>
 
-                <button className="footer__button">
+                <FooterButton>
                 Create
 
-                </button>
+                </FooterButton>
 
-                <button className="footer__button">
+                <FooterButton>
                 Profile
 
-                </button>
-            </nav>
+                </FooterButton>
+            </FooterContainer>
         </footer>
     )
 }
 
-export default Footer
\ No newline at end of file
+const FooterContainer = styled.nav`
+position: fixed;
+bottom: 0px;
```

Comment:
no need to explicitly say "px" here. just a ***zero*** is enough

```suggestion
bottom: 0;
```